flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/operators/multipleinput/input/InputSelectionHandlerTest.java

```diff
@@ -41,7 +42,11 @@ public void testWithSamePriority() {
                         new InputSpec(3, 0, createTwoInputOperatorWrapper("input3"), 1),
                         new InputSpec(4, 0, createTwoInputOperatorWrapper("input4"), 2),
                         new InputSpec(5, 0, createOneInputOperatorWrapper("input5"), 1));
-        InputSelectionHandler handler = new InputSelectionHandler(inputSpecs);
+        InputSelectionHandler handler =
+                new InputSelectionHandler(
+                        inputSpecs.stream()
+                                .map(InputSpec::getMultipleInputSpec)
+                                .collect(Collectors.toList()));
```

Comment:
We can still keep the `List<InputSpec>` constructor of `InputSelectionHandler` to avoid verbose type conversion here and there.

Reply:
Due to the Java collection generic erasure, we cannot keep the original constructor.

Reply:
You can have a static method for the `List<InputSpec>`, for example: `InputSelectionHandler.fromInputSpecs(List<InputSpec>)`.